packages/box/src/handlers/sessions.ts

```diff
@@ -0,0 +1,87 @@
+import express, { Request, Response } from "express";
+import {
+    handleErrors,
+    BadRequestError,
+    InternalServerError,
+    sessionSignRequest,
+    SessionSignRequest,
+} from "@edenos/common";
+import { arrayToHex } from "eosjs/dist/eosjs-serialize";
+
+import logger from "../logger";
+import { eosJsonRpc, eosDefaultApi } from "../eos";
+import { edenContractAccount } from "../config";
+import { TaposManager } from "../sessions";
+
+export const sessionHandler = express.Router();
+
+const taposManager = new TaposManager(eosJsonRpc);
+taposManager.init();
+
+sessionHandler.post("/sign", async (req: Request, res: Response) => {
+    try {
+        const { body } = req;
+        if (!body) {
+            throw new BadRequestError(["missing session sign data"]);
+        }
+
+        const parsedRequest = sessionSignRequest.safeParse(JSON.parse(body));
+        if (parsedRequest.success !== true) {
+            throw new BadRequestError(parsedRequest.error.flatten());
+        }
+
+        const requestData: SessionSignRequest = parsedRequest.data;
+        return await signSessionRequest(requestData, res);
+    } catch (error) {
+        logger.error(`sessionHandler: ${error.message}`);
+        return handleErrors(res, error);
+    }
+});
+
+const signSessionRequest = async (
+    sessionSignRequest: SessionSignRequest,
+    res: Response
+) => {
+    const trx = prepareExecSessionTrx(sessionSignRequest);
+    const signedTrxResult = await signTrx(trx);
+    logger.info(
+        `signed [execsession] for account:${sessionSignRequest.edenAccount}
+        sequence:${sessionSignRequest.sequence}`
+    );
+    return res.json(signedTrxResult);
+};
+
+const prepareExecSessionTrx = (sessionSignRequest: SessionSignRequest) => {
```

Comment:
This is missing the no-op action which pays for the transaction

Reply:
But the transaction is a single `execsession` being signed by the server, no? That being the case, do we need a noop serverpay action?

Reply:
@tbfleming I addressed this in a new commit, can you please give another round of reviews?